app/routes/users/components/PhotoConsents.tsx

```diff
@@ -52,9 +52,15 @@ const ConsentManager = ({
         <ConfirmModal
           closeOnConfirm={true}
           title={`Trekke bildesamtykke på ${presentableDomain}`}
-          message={`Er du sikker på at du vil trekke bildesamtykket ditt for ${toReadableSemester(
-            consent,
-          )} på ${presentableDomain}? Dersom du ønsker å fjerne noen spesifikke bilder, kan du i stedet sende en e-post til pr@abakus.no med informasjon om hvilke bilder du vil fjerne.`}
+          message={
+            <span>
+              Er du sikker på at du vil trekke bildesamtykket ditt for
+              {toReadableSemester(consent)} på {presentableDomain}? Dersom du
+              ønsker å fjerne noen spesifikke bilder, kan du i stedet sende en
+              e-post til <a href="mailto:pr@abakus.no">pr@abakus.no</a> med
+              informasjon om hvilke bilder du vil fjerne.
+            </span>
```

Comment:
Seems like the message gets wrapped in a `span` tag anyways. I guess it does not hurt, but any reason for adding it here as well?

Reply:
You're right, thats unnecessary! Removed it now - tested that it still work hehe